synfig-studio/src/gui/trees/layertree.h

```diff
@@ -176,12 +160,6 @@ class LayerTree : public Gtk::Table
 
 	void on_selection_changed();
 
-	void on_dirty_preview();
-
-	void on_amount_value_changed();
-
-	void on_blend_method_changed();
```

Comment:
This was the only reason of this last force-push: I forgot to erase its declarations